homeassistant/components/kaleidescape/remote.py

```diff
@@ -0,0 +1,88 @@
+"""Kaleidescape remote."""
+from __future__ import annotations
```

Comment:
Please limit pull requests to a single platform.  The smallest platform is preferred for new integraitons


https://developers.home-assistant.io/docs/creating_component_code_review?_highlight=single&_highlight=platform#5-make-your-pull-request-as-small-as-possible

Reply:
Removed.